_posts/2023-05-11-maintaining-history-telling-stories

```diff
@@ -2,8 +2,24 @@
 ##### Liam Lynch
 ##### Interned February 1 - May 3

```

Comment:
We can take out lines 1-3. If we want to include the dates of the internship, can we add it to the end of the post?

Reply:
Removed those lines.